examples/PythiaConnector.cpp

```diff
@@ -46,72 +46,48 @@ PythiaConnector::PythiaConnector(const char* fname) : m_store(podio::EventStore(
   m_store.setReader(&m_reader);
 }
 
-papas::ListParticles PythiaConnector::makePapasParticlesFromGeneratedParticles(const fcc::MCParticleCollection* ptcs) {
+void PythiaConnector::makePapasParticlesFromGeneratedParticles(const fcc::MCParticleCollection* ptcs,
```

Comment:
For now let's leave it like this. 
At some point, we should change that on both the c++ and python sides. can you create an issue with low priority to keep track of that?